src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -606,9 +606,6 @@
   <data name="PowerRename_Toggle_ContextMenu.Header" xml:space="preserve">
     <value>Show PowerRename in</value>
   </data>
-  <data name="PowerRename_Toggle_ContextMenu.Description" xml:space="preserve">
```

Comment:
Isn't this still needed for Windows 10 devices?

Reply:
"needed"is a difficult word. Up to now, it's also shown in Win.11 and that's just wrong. And (to quote some unnamed persons) "PowerToys is intented for power-users", right? I'd expect power users know how to show the extended menu.

Reply:
I'd leave this in for now as it's a regression.. We can open a new issue and make a helper class that hides this message when  on W11. Would be useful for other things (in the future as well)?

Reply:
That helper is a fine idea too - we already have one. (for example to hide a "rounded corners" settings for FZ.)